descriptor/parser_test.go

```diff
@@ -350,6 +350,27 @@ func TestParseScriptExpressionWpkh(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "wpkh_xpub_no_fingerprint",
+			args: args{
+				descriptor: "wpkh(xpub661MyMwAqRbcFFzgbS7PZzrLLXNYmno5FN7aYMceX7HcRgot6DUnPWn8z8C2EAcqiQ9QBmsWkVmhvMjsrwsMexwiqcW1mdyMZDspQqv6SUQ/1/*)",
+				topLevel:   true,
+			},
+			validate: func(wallet Wallet) error {
+				scriptHexExpected := "0014881e461e5a3d4114011aedb9242fa4b05d0a9142"
+				scripts, err := wallet.Script(nil)
+				if err != nil {
+					return err
+				}
+
+				if hex.EncodeToString(scripts[0].Script) != scriptHexExpected {
+					return errors.New("unexpected script gen")
+				}
+
+				return nil
+			},
+			wantErr: false,
+		},
```

Comment:
let's add also a test without child keys? ie. only the child xpub passes

Reply:
Done